src/mem/qos/mem_ctrl.hh

```diff
@@ -60,7 +60,7 @@ class MemCtrl : public ClockedObject
 {
   public:
     /** Bus Direction */
-    enum BusState { READ, WRITE };
+    enum BusState { READ, WRITE, ReadBeforeWrite };
```

Comment:
Hmm... this clearly doesn't match the style.

Also "read before write" isn't a *bus state*. It's something else.